docs/java/index.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Java ♡ Ethereum
+lang: en-US
+sidebar: auto
+sidebarDepth: 0
+
+---
+# Java ♡ Ethereum
```

Comment:
Same here - could we update this to "Ethereum for Java Developers"?

Reply:
Fixed!